src/modules/job-manager/jobtap.c

```diff
@@ -410,22 +410,13 @@ int jobtap_load (struct jobtap *jobtap,
         }
     }
 
-    /*  Always unload current plugin before loading next plugin.
```

Comment:
commit message spelling "remvoe", "jobta_load()", and s/when/with in last sentence